CauseContract.sol

```diff
@@ -34,7 +37,7 @@ contract CauseContract {
     // donor proportion tracking
     mapping(address => uint256) public donorTotals;
     
-    mapping(address => bool) public addressDonated;
+    // mapping(address => bool) public addressDonated;
```

Comment:
what is this comment for?

Reply:
I kept the original code in the form of comment for comparison with the changed code.
Here I changed mapping to array for gas saving. Calm, I just don't want to cause any confusion